web/src/js/__tests__/ducks/tutils.js

```diff
@@ -7,3 +7,30 @@ export function createStore(parts) {
         applyMiddleware(...[thunk])
     )
 }
+
+export function TFlow(intercepted=false, marked=false, modified=false) {
+    return {
+        intercepted : intercepted,
+        marked : marked,
+        modified: modified,
+        id: "foo",
+        request: {
```

Comment:
Sounds good, in this way we don't have to worry about what attributes should be add to `tflow`. I would like to merge this PR right away and implement the `_tflow.js`. Then come back for the left components in FlowTable later.